code-reviews/recipes/Terraform.md

```diff
@@ -1,18 +1,23 @@
-# Terraform Code Reviews
+## Code Reviews
+### Style Guide
+- Adhere to the [hashicorp style guide](https://www.terraform.io/docs/configuration/style.html)
```

Comment:
Some things in this section seem to fit better in the check list.
Suggestion:  keep the following in this section (with suggested rewording to fit the other recipes)

[CSE](../../CSE.md) developers follow the [hashicorp style guide](https://www.terraform.io/docs/configuration/style.html).

[CSE](../../CSE.md) projects should check Terraform scripts with automated tools.

Move the other items to the check list portion. Suggestions for wording below:

- [ ] Are all providers used in the terraform scripts [versioned](https://www.terraform.io/docs/configuration/providers.html#provider-versions) to prevent breaking changes in the future?
- [ ] Are modules split into separate .tf files where appropriate?
- [ ] Does the repository contain a `README.md` describing the architecture provisioned

(Note: not sure about the last check point, please reword if I misunderstood)

Reply:
updated based on the comment.